app/assets/javascripts/actions/campaign1_actions.js

```diff
@@ -0,0 +1,33 @@
+import {
```

Comment:
This file needs a better name. For now, I suggest `campaign_view_actions.js`.

`campaign_actions.js` is ultimately probably the right name, but we'd first need to rename the current file of that name to something like `course_campaign_actions`, since it's actions related to the campaigns of a particular course rather than the campaign itself.

Reply:
I going to change that file name.